webpack.config.client.js

```diff
@@ -46,7 +46,9 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       // need full URL for dev server & HMR: https://github.com/webpack/docs/wiki/webpack-dev-server#combining-with-an-existing-server
       publicPath: START_DEV_SERVER
         ? `http://localhost:${webpackDevServerPort}/`
-        : `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}/`,
+        : `https://${
```

Comment:
Running `npm run build && npm start` the js assets cant resolve `https://undefined/include/articles/public/static/js/main-d0ae3f07.aa09710c.js`

Reply:
Whoops, I only tried `npm run dev` locally, not production build. Should be fixed now!